src/player.py

```diff
@@ -0,0 +1,7 @@
+class Player():
+    def __init__(self, name, token="X"):
+        self.name = name
```

Comment:
Does the player need a name just yet? Or is this more of a type? like "human" or "computer"?

Reply:
Ah you're right, the player doesn't need a name yet as we adjusted the story for this one. I had remnants of the old story in my head still and that's why I added the name functionality. Mostly to prepare for the future. 

Again, might have jumped the gun on it.